.github/workflows/main-cmake-spc.yml

```diff
@@ -199,6 +199,66 @@ jobs:
         run: ctest . --parallel 2 -C Debug -V
         working-directory: ${{ runner.workspace }}/build
 
+  build_v1_14:
+    name: "gcc DBG v1.12 default API"
```

Comment:
v1.12 -> v1.14, though it looks like we also should have a v1.16 default API build as well?

Reply:
Oh I thought updated this, must have forgot to save